frontend/simple/js/idb-flume-log.js

```diff
@@ -0,0 +1,192 @@
+import localforage from 'localforage'
+import { toHash, makeEntry } from '../../../shared/functions'
+import Append from 'append-batch'
+import Obv from 'obv'
+const storeName = 'EventLogStorage'
+const name = 'Group Income'
+
+// Generate a log
+export default async function log (vuex) {
+  // Initialize localforage to Index
+  // https://www.html5rocks.com/en/tutorials/offline/quota-research/#toc-overview
+  localforage.config({
+    driver: localforage.INDEXEDDB,
```

Comment:
The were not drivers available for these choices in localForage therefore I arrived at IndexDB